posthog/cdp/templates/braze/template_braze.py

```diff
@@ -0,0 +1,95 @@
+from posthog.cdp.templates.hog_function_template import HogFunctionTemplate
+
+
+template: HogFunctionTemplate = HogFunctionTemplate(
+    status="beta",
+    id="template-braze",
+    name="Send events to Braze",
+    description="Send events to Braze",
+    icon_url="/static/services/braze.png",
+    hog="""
+let getEndpoint := () -> {
+  'US-01': 'https://rest.iad-01.braze.com',
+  'US-02': 'https://rest.iad-02.braze.com',
+  'US-03': 'https://rest.iad-03.braze.com',
+  'US-04': 'https://rest.iad-04.braze.com',
+  'US-05': 'https://rest.iad-05.braze.com',
+  'US-06': 'https://rest.iad-06.braze.com',
+  'US-08': 'https://rest.iad-08.braze.com',
+  'EU-01': 'https://rest.fra-01.braze.eu',
+  'EU-02': 'https://rest.fra-02.braze.eu',
+}[inputs.brazeEndpoint]
+
+let getPayload := () -> [{
+  'attributes': inputs.attributes,
+  'events': [inputs.event]
+}]
+
+let res := fetch(f'{getEndpoint()}/users/track', {
+  'method': 'POST',
+  'headers': {
+    'Authorization': f'Bearer {inputs.apiKey}',
+    'Content-Type': 'application/json'
+  },
+  'body': getPayload()
+})
+
+if (res.status >= 200 and res.status < 300) {
+  print('Event sent successfully!')
+} else {
+  throw Error(f'Error sending event: {res.status} {res.body}')
+}
+""".strip(),
+    inputs_schema=[
+        {
+            "key": "brazeEndpoint",
+            "type": "choice",
+            "label": "Braze REST Endpoint",
+            "description": "The endpoint identifier where your Braze instance is located, see the docs here: https://www.braze.com/docs/api/basics",
+            "choices": [
```

Comment:
Not a biggy but the values for these could just be the endpoint instead of doing the mapping. Thats what I did for most other similar cases

Reply:
yes, that makes it a lot better 👍